docker/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Copyright DataStax, Inc.
+# Please see the included license file for details.
+FROM --platform=${BUILDPLATFORM} maven:3.6.3-adoptopenjdk-8 as builder
```

Comment:
What the "--platform=${BUILDPLATFORM}" does here is cause this stage to always run for the platform the build is happening on regardless of the target platform/architecture. For example, if we are running the build on amd64 but target arm64, this stage will run for the native platform of amd64. We can do this because the JAR produced by this stage is platform independent and building it without going through an emulator is faster.

Reply:
Can we move this write-up ^ into a comment in the Dockerfile?